n2y/plugins/jinjarenderpage.py

```diff
@@ -142,19 +177,25 @@ def render_from_string(source, context=None, environment=None):
 class JinjaRenderPage(Page):
     def __init__(self, client, notion_data):
         super().__init__(client, notion_data)
-        self.client.plugin_data['jinjarenderpage'] = {
-            'environment': _create_jinja_environment(),
-        }
+        if 'jinjarenderpage' not in client.plugin_data:
+            client.plugin_data['jinjarenderpage'] = {}
+        if self.notion_id not in client.plugin_data:
+            client.plugin_data['jinjarenderpage'][self.notion_id] = {
+                'environment': _create_jinja_environment(),
+                'database_cache': {},
```

Comment:
Let's remove the "database_cache" here. We already cache at the network request level (see https://github.com/innolitics/n2y#cache).

https://wiki.c2.com/?PrematureOptimization=